lib/kino/explorer.ex

```diff
@@ -184,6 +187,15 @@ defmodule Kino.Explorer do
     |> DataFrame.to_columns()
   end
 
+  defp compute_frequencies?(series) do
+    case Series.dtype(series) do
+      {:list, dtype} -> 
+        numeric_type?(dtype)
```

Comment:
For backwards compatibility we should return `false` if the `dtype` is `:integer` or `:float`

Reply:
I am confused about backward compatibility for :integer and float false. {list, numeric} is supported in explorer 0.7.3+ . :integer is converted to {s, 64} and :float to {f, 64} ? Can you explain little more..

Reply:
In previous versions this conversion does not exist and the types are `{:list, :integer}` and `{:list, :float}`. Both are not supported. You can reproduce the bug by running your fixed version of `Kino_explorer` with the currently released version of `explorer 0.7.2`, and you will see that the problem persists. Returning `false` for these legacy types solves the problem and covers these cases.

Reply:
For = 0.7.2 it fails for  list of any numeric type - and :float is converted to {:f, 64}. So check for {:list, :float} does not work.
Before < 0.7.2  {:list, _} is not supported ? `Series.from_list([[1], [2]], dtype: {:list, :float}).dtype` fails with unsupported datatype.

 If we want backward compatibility for = 0.7.2, a better way would be to check if {:s, 8} is a member of Explorer.Shared.dtypes() outside `for` loop and pass this to `compute_summaries?/2` ?

Reply:
Yes, it's a good approach. As for versions prior to `0.7.2`, the error is directly in Explorer and we won't cover it here.